Apps/Playground/CMakeLists.txt

```diff
@@ -152,6 +169,7 @@ if(APPLE)
             XCODE_ATTRIBUTE_SWIFT_VERSION "4.0"
             XCODE_ATTRIBUTE_SWIFT_OBJC_BRIDGING_HEADER "${CMAKE_CURRENT_LIST_DIR}/iOS/LibNativeBridge.h"
             XCODE_ATTRIBUTE_LD_RUNPATH_SEARCH_PATHS "@executable_path/Frameworks"
+            XCODE_ATTRIBUTE_FRAMEWORK_SEARCH_PATHS "$(inherited) $(SDKROOT)$(SYSTEM_LIBRARY_DIR)/Frameworks"
```

Comment:
Is this newly needed on iOS?

Reply:
I don't think it's new. This is an additional fix for the simulator builds without it the simulator doesn't link built-in libraries. 

![CleanShot 2024-08-27 at 13 53 13@2x](https://github.com/user-attachments/assets/528fe652-1ffb-40b7-b09f-0b4c68746bdd)